superset/config.py

```diff
@@ -109,6 +109,7 @@
 # },
 FAVICONS = [{"href": "/static/assets/images/favicon.png"}]
 
+LOADERS = [{"src": "/static/assets/images/loading.gif"}]
```

Comment:
Perhaps this is a bit of a side mission for another PR, or it could be part of this, but I notice we have the FAVICON and LOADERS up here. APP_ICON is waaaaay down below, but maybe they should live together in the config?

LOADERS could also probably use a comment explaining what it is, for anyone learning what they can config.

Is there a reason it's an array? Can you use more than one? If it's just a single GIF, it should probably be more singular like APP_ICON. And to be crazy pedantic, maybe it should just be "LOADER" rather than "LOADERS"